docs/okeps/okep-5674-dpu-healthcheck.md

```
@@ -0,0 +1,169 @@
+
+Only applicable for Trusted DPU Architecture. Zero-trust will be handled later.
+
+## OVN-Kubernetes Version Skew
+
+Targeting version 1.2.
+
+## Alternatives
+
+Instead of using the node condition to indicate the CNI is down, cluster-manager could listen for the lease expiry and
+then taint the node. We have tried tainting the node in the past, and it led to a bunch of issues causing us to revert it:
+https://github.com/ovn-kubernetes/ovn-kubernetes/pull/2459. The SDN should refrain from tainting nodes, as it can cause
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)